core/go/internal/domainmgr/private_smart_contract.go

```diff
@@ -477,12 +488,14 @@ func (dc *domainContract) EndorseTransaction(dCtx components.DomainContext, read
 	if err != nil {
 		return nil, err
 	}
-	return &components.EndorsementResult{
+	log.L(dCtx.Ctx()).Infof("Endorse transaction returning endorsement result with payload length %d: %+v", len(res.Payload), res.Payload)
```

Comment:
Not sure the length of the payload is the most significant summary item to return here

Reply:
Agreed - similar to my previous comment, I think I was using this for a `>0` or not check in the logs. I'll make a note to review our logging towards the end of the review process (possibly post-merge)